packages/cli/src/main.ts

```diff
@@ -19,12 +19,19 @@ timingApi.mark("PRE_IMPORT_IMPERATIVE");
 import { IImperativeConfig, Imperative } from "@zowe/imperative";
 import { Constants } from "./Constants";
 import { inspect } from "util";
+import { homedir } from "os";
 
 // TODO(Kelosky): if we remove this, imperative fails to find config in package.json & we must debug this.
 const config: IImperativeConfig = {
-    configurationModule: __dirname + "/imperative"
+    configurationModule: "imperative.js"
 };
 
+const logConfig: IImperativeConfig = {
+    name: Constants.DISPLAY_NAME
+};
+
+const homeDir = homedir();
```

Comment:
I'm curious what the purpose of this getting the `homedir` here is.

Reply:
At one point it was being used to do imperative logger initializations. I think it can be removed.

Reply:
Nothing, anymore.